src/input/command.zig

```diff
@@ -538,6 +538,11 @@ fn actionCommands(action: Action.Key) []const Command {
                 .title = "Close Other Tabs",
                 .description = "Close all tabs in this window except the current one.",
             },
+            .{
+                .action = .{ .close_tab = .right },
+                .title = "Close Tabs to the Right",
+                .description = "Close every tab to the right of the current one.",
```

Comment:
I wonder if it would be a good idea to make the commend and description the same:

```zig
/// Close all tabs to the right of the current tab.
```